tests/component-tests/pager/js/PagedTableTests.js

```diff
@@ -222,7 +269,7 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
                 options: {
                     moduleSource: {
                         func: "fluid.tests.tooltipModuleSource",
-                        args: "{pager}"
+                        args: ["{pager}", [2, 3, 4]]
```

Comment:
But, there cannot be any instance where a sequence of page sizes is reusable. I mean it's a test data.